canonical-web-and-design/jaas.ai
# Dedupe models

## Done

- Transform charm and bundle definitions into classes.
- Remove duplication of attributes.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Have a poke around pages that show charms/bundles. Everything should be unchanged.
